Dockerfile

```diff
@@ -1,30 +1,23 @@
-FROM ubuntu:trusty
+FROM ubuntu:bionic
 MAINTAINER Zach Halpern <zahalpern+github@gmail.com>
 
-RUN apt-get update && apt-get install -y software-properties-common
-RUN apt-add-repository ppa:ubuntu-sdk-team/ppa
-RUN add-apt-repository -y ppa:smspillaz/cmake-master
 RUN apt-get update && apt-get install -y\
-  build-essential g++\
+  build-essential\
   cmake\
```

Comment:
Do we still need this?
I think we use the bundled cmake now?

Reply:
We still need to actually install cmake to use cmake.